lib/msal-browser/docs/logout.md

```diff
@@ -82,7 +82,13 @@ const currentAccount = msalInstance.getAccountByHomeId(homeAccountId);
 await msalInstance.logoutPopup({
     account: currentAccount,
     postLogoutRedirectUri: "https://contoso.com/loggedOut",
-    mainWindowRedirectUri: "https://contoso.com/homePage"
+    mainWindowRedirectUri: "https://contoso.com/homePage",
+    popupConfiguration: {
+        height: 100,
```

Comment:
Values of 100/100/100/100 are great for a sample or test, but the code should likely enforce some sane defaults.  For instance, 1/0/9999/9999 shouldn't be a valid popup config.  Considerations: popup should be smaller than screen/parent window size, trying to position off-screen should likely be invalid as well.  If we think the config is bad, we can reject it.  Documentation should be updated to reflect the approach we take for this.

Reply:
We have default height width defined already: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/f9d2bdbab54f39cb32cd7f82acb5647a296db493/lib/msal-browser/src/utils/BrowserConstants.ts#L25-L29

And top/left is dynamically calculated based on the window. I agree there are some non-sensical values we could/should protect against, but I think we want to be careful to be too opinionated (given an application can open its own popup window with whatever dimensions it wants, in theory).

Reply:
Any suggestions for these defaults? I recognize 1/1/1/1 would be virtually useless, but agree that we don't want to be too opinionated

Reply:
Thanks for the suggestions. Have decided to log errors and use defaults when popup window is out of bounds, non-sensical values are used, or window is larger than screen/parent window size. Have not set other minimums or maximum, as we do not want to be too opinionated, and it should be up to the developer to set a popup window size that is reasonable for their app and users to use.